courses/machine_learning/deepdive2/time_series_prediction/labs/2_feature_engineering.ipynb

```diff
@@ -0,0 +1,1298 @@
+{
```

Comment:
<p>Is SUN what they should be looking into? Maybe a little comment for clarity?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
<p>Just removed this section as I don' think it's needed :)</p>